### ISSUE TITLE
Remove void keyword before the function keyword

### ISSUE THREAD

**Issue by rilysh** [davidchambers/string-format#53]
I saw at the first line that there's a void keyword before the function keyword, I understood the use case but it's nowadays an inappropriate way to use the function. Some eslint config (such as airbnb) avoid using void keyword, during its no effect feature.

**Comment by davidchambers** [comment]
You're right. Much about this project could be improved now that we have ES modules and other new language features. :)
